src/views/pages/models.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ModelStore from '../../store/model-store.js';
+import CreateModelForm from '../components/create-model-form.jsx';
+import { getAllModels } from '../../lib/api-client'
+export default class Models extends ListenerComponent {
+
+    constructor() {
+        super();
+        getAllModels
```

Comment:
Why is this called here?  Didn't you call it when you clicked the button to go to this page?